### ISSUE TITLE
Max retries exceeded with url: http://localhost:8000/api/auto/update assetfinder

### ISSUE THREAD

**Issue by 0xdej** [R-s0n/ars0n-framework#23]
Hi @R-s0n,

I was following your tutorial in youtube and i successfully run the wildfire script same in the video but the scan doesnt finish because of this error below

[!] Something went wrong!  Exception: HTTPConnectionPool(host='127.0.0.1', port=8080): Max retries exceeded with url: http://localhost:8000/api/auto/update (Caused by ProxyError('Cannot connect to proxy.', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f413c1024d0>: Failed to establish a new connection: [Errno 111] Connection refused')))
[-] Running Assetfinder against domain.com
[!] Something went wrong!  Exception: HTTPConnectionPool(host='127.0.0.1', port=8080): Max retries exceeded with url: http://localhost:8000/api/auto/update (Caused by ProxyError('Cannot connect to proxy.', NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f413c100bd0>: Failed to establish a new connection: [Errno 111] Connection refused')))
[-] Running Get All URLs against domain.com

**Comment by quyenvuongu** [comment]
I have the same issue with fresh kali VM.

**Comment by R-s0n** [comment]
Shoot, sorry about that!  I made the proxy mandatory when I was testing the Alpha release and must have forgotten to remove it.  

That error message is caused because it's trying to route all the HTTP calls through Burp on 127.0.0.1:8080.  This error occurs when Burp isn't running, but this shouldn't be mandatory.  I just pushed a commit that *should* fix it.  

A stable Alpha release will be available in the next week or so!
